plugins/modules/nagios.py

```diff
@@ -90,7 +90,8 @@
         services running on it.) To schedule downtime for all O(services) on particular host use keyword V(all), for example
```

Comment:
Can you change the above documentation

> You can specify multiple services at once by separating them with commas, for example O(services=httpd,nfs,puppet).

to something like

> Before community.general 11.2.0, one could specify multiple services at once by separating them with commas, for example O(services=httpd,nfs,puppet). Since community.general 11.2.0, there can be spaces around the commas, and an actual list can be provided.

and move it to the bottom of the list?

Reply:
done